gen/aircraft.py

```diff
@@ -777,7 +786,9 @@ def _setup_group(self, group: FlyingGroup, for_task: Type[Task],
             divert=divert,
             # Waypoints are added later, after they've had their TOTs set.
             waypoints=[],
-            intra_flight_channel=channel
+            intra_flight_channel=channel,
+            bingo_fuel=flight.flight_plan.bingo_fuel,
+            joker_fuel=flight.flight_plan.joker_fuel
```

Comment:
No need for these to be stored directly in the flight, since Flight.flight_plan is already there.

Reply:
(ignore me, this isn't the class I thought I was reading)